runtime/queries/apex/highlights.scm

```diff
@@ -36,43 +32,46 @@
 (super) @function.builtin
 
 ;; Annotations
-
 (annotation
   name: (identifier) @attribute)
 
 ;; Types
-
 (interface_declaration
   name: (identifier) @type)
 
 (class_declaration
   name: (identifier) @type)
 
 (class_declaration
-  (superclass) @type) 
+  (superclass) @type)
 
 (enum_declaration
   name: (identifier) @type)
 
 (enum_constant
   name: (identifier) @constant)
 
-(type_arguments "<" @punctuation.delimiter)
-(type_arguments ">" @punctuation.delimiter)
+(type_arguments
+  "<" @punctuation.delimiter)
+
+(type_arguments
+  ">" @punctuation.delimiter)
 
 ((field_access
   object: (identifier) @type))
 
 (field_access
   field: (identifier) @property)
 
-((scoped_identifier
-  scope: (identifier) @type)
- (#match? @type "^[A-Z]"))
+(
```

Comment:
Can we keep the "collapsed" opening parens (since we collapse the corresponding closing parens)?

Reply:
We can.